index.js

```diff
@@ -408,7 +413,7 @@ function normalizeRoot (root) {
   return root
 }
 
-function checkRootPathForErrors (fastify, rootPath) {
+function checkRootPathForErrors (fastify, rootPath, skipExistenceCheck) {
```

Comment:
I am think is it necessary to skip the check of `root` path?
You should never use a invalid root path, defaulting to `cwd` is a trade-off for security reason.

All of your served file should be contained within to the `root` and prevent path escape.

Reply:
@climba03003 Thank you for your feedback on the security implications. Before implementing changes, I would like to confirm the approach that we remove the `skipExistenceCheck` parameter from the `checkPath` and `checkRootPathForErrors` functions since we need perform full validation of the path, including existence checks.

Reply:
You can skip for cwd, but you must provide a valid root.
No matter other setting.